modules/identity-provider-configuring-apache-request-header.adoc

```diff
@@ -212,8 +189,8 @@ RequestHeader unset X-Remote-User
 [id="requestheader-master-ca-config_{context}"]
 ===== Configuring the master
 
-The `identityProviders` stanza in the
-*_/etc/origin/master/master-config.yaml_* file must be updated as well:
+Update The `identityProviders` stanza in the
```

Comment:
There is no such file, see the rest of the Request Header Identity Provider guide to see how stuff is being configured now.

Reply:
I see the Custom Resource defined [here](https://docs.openshift.com/container-platform/4.2/authentication/identity_providers/configuring-request-header-identity-provider.html#identity-provider-request-header-CR_configuring-request-header-identity-provider), which I assume is the proper configuration for 4.x. I can't seem to locate the specific YAML file this is defined in though. What file should I specify if `/etc/origin/master/master-config.yaml` no longer exists?

How does this updated `identityProviders` stanza look?

```
identityProviders:
  - name: requestheaderidp 
    mappingMethod: claim 
    type: RequestHeader
    requestHeader:
      challengeURL: "https://<namespace_route>/challenging-proxy/oauth/authorize?${query}" 
      loginURL: "https://<namespace_route>/login-proxy/oauth/authorize?${query}" 
      ca: 
        name: ca-config-map
      clientCommonNames: 
      - my-auth-proxy
      headers: 
      - X-Remote-User
```
Anything to add/edit/remove?

Reply:
You could leave out `mappingMethod: claim` as that's the default, but it LGTM